web/impact/impact/settings.py

```diff
@@ -279,6 +285,80 @@ class Base(Configuration):
     IMAGE_RESIZE_HOST = "https://dl4fx6jt7wkin.cloudfront.net"
     IMAGE_RESIZE_TEMPLATE = "/fit-in/500x500/media/{}"
 
+    LOGGING = {
+        'version': 1,
+        'disable_existing_loggers': True,
+        'filters': {
+            'require_debug_false': {
+                '()': 'django.utils.log.RequireDebugFalse'
+            }
+        },
+        'formatters': {
+            'detail': {
+                'format': LOG_FORMAT,
+                'datefmt': '%Y-%m-%dT%H:%M:%S',
+            },
+        },
+        'handlers': {
+            'null': {
+                'class': 'logging.NullHandler',
+            },
+            'file': {
+                'level': 'INFO',
+                'class': 'logging.handlers.RotatingFileHandler',
+                'maxBytes': 1024 * 1024 * 5,
+                'backupCount': 5,
+                'formatter': 'detail',
+                'filename': 'debug.log',
+            },
+            'mail_admins': {
+                'level': 'ERROR',
+                'filters': ['require_debug_false'],
+                'class': 'django.utils.log.AdminEmailHandler'
```

Comment:
Adding `mail_admins` handling seems outside the scope of the ticket.

Reply:
the `LOGGING` parameter must be defined with all the expected keys, including `mail_admins`, or we face errros

Reply:
Interesting. If we agree that we don't mind adding `mail_admins` here (and we figure out in advance of deployment roughly what we expect the message volume to be), I can live with adding it.

Otherwise, re disabling it, a couple possible leads I found:
 * [disabling the automatic logging config](https://docs.djangoproject.com/en/1.11/topics/logging/#disabling-logging-configuration)
* using `logging.NullHandler` (example)[https://docs.djangoproject.com/en/1.11/topics/logging/#django-security]
* a few [ideas from Lincoln Loop](https://lincolnloop.com/blog/disabling-error-emails-django/)